app/mattermost.js

```diff
@@ -90,6 +92,7 @@ export default class Mattermost {
 
         setJSExceptionHandler(this.errorHandler, false);
         setNativeExceptionHandler(this.nativeErrorHandler, false);
+        setSystemEmojis(EmojiIndicesByAlias);
```

Comment:
this should be done once the store has been rehydrated.

Reply:
Does it need to be? This doesn't come from the store, and it's not written there either

Reply:
What is the redux action doing then?

Reply:
It's not a true action, it's not dispatching anything just setting the system emojis to a global (not in the store since they're more of a constant)